src/components/auth/LoginForm/index.jsx

```diff
@@ -52,6 +65,9 @@ export const LoginForm = () => {
                             className="w-full border-gray-100 rounded-md placeholder:text-gray-500"
                             placeholder="Contraseña"/>
                     </div>
+                    {
+                        showErrorMessage && <p className="text-red-500 text-sm font-medium mb-4">{errorMessage}</p>
```

Comment:
If you use something more than once in different components, creating a new reusable component may be a good idea

In this case let's make an `ErrorMessage` component in `/src/components/forms/ErrorMessage`

Keep in mind margin at bottom shouldn't be a default